i18n/en/docusaurus-plugin-content-docs/current/reference/slices-segments.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Slices and segments
+sidebar_position: 2
+pagination_next: reference/public-api
+---
+
+# Slices and segments
+
+## Slices
+
+Slices are the second level in the organizational hierarchy of Feature-Sliced Design. Their main purpose is to group code by its meaning for the product, business or just the application.
+
+The names of slices are not standardized because they are directly determined by the business domain of your application. For example, a photo gallery might have slices `photo`, `create-album`, `gallery-page`. A social network would require different slices, for example, `post`, `add-user-to-friends`, `news-feed`.
+
+Closely related slices can be structurally grouped in a directory, but they should exercise the same isolation rules as other slices — there should be **no code sharing** in that directory.
+
+![Features "compose", "like" and "delete" grouped in a directory "post". In that directory there is also a file "some-shared-code.ts" that is crossed out to imply that it's not allowed.](/img/graphic-nested-slices.svg)
+
+The layers Shared and App don't contain segments. That is because Shared should contain no business logic at all, hence has no meaning for the product, and App should contain only code that concerns the entire application, so no splitting is necessary.
```

Comment:
The layers Shared and App don't contain **segment**
 
 -> 
 
 The layers Shared and App don't contain **slices**

Reply:
Good catch

Reply:
As i get it, we will add custom **slices** in FSD Extensions?

Reply:
Custom "slices"? Slices were never standardized, so in a way they are always custom. Not sure what you're referring to

Reply:
custom segments which became too big to treat and maintain them as segments, so they evolve into slices

Reply:
I don't think segments can evolve into slices. Segments and slices are not only a structural grouping, but also a semantic grouping, therefore, they are not interchangeable.

Could you give an example please?

Reply:
shared/api -
  - if u have simple api routes — it can be just segment
  - if u also have some sort of model (usually tokens and headers managment) and some libs (`isAuthError(code)`) it can became slice
  - (*edge case) if u have a more than one backend, u should separate them into their own slices, so shared/api became layer